critiquebrainz/frontend/templates/footer.html

```diff
@@ -22,7 +22,7 @@
       <ul>
         <li><a href="https://critiquebrainz.readthedocs.io/api.html">{{ _('Web API') }}</a></li>
         <li><a href="https://github.com/metabrainz/critiquebrainz">{{ _('Source code') }}</a></li>
-        <li><a href="ftp://ftp.musicbrainz.org/pub/musicbrainz/critiquebrainz/">{{ _('Data dumps') }}</a></li>
+        <li><a href="http://ftp.musicbrainz.org/pub/musicbrainz/critiquebrainz/">{{ _('Data dumps') }}</a></li>
```

Comment:
Nit: Maybe `http://data.metabrainz.org/pub/musicbrainz/critiquebrainz/` instead it is the main server? 

Although the ftp one also pointed to the mirror so its probably fine either way.

Reply:
yeah, I just updated the protocol and left the domain. I note that on https://listenbrainz.org/data/ we call this host the "main download site (USA)", so I guess for some definition it's also a main site. I'm happy to update it if you think it's important